### PR TITLE
Refactoring components without BEM CSS pattern

### PR DESCRIPTION
**Pull request by richardkevin**

### CODE REVIEW

**Review by vierno — CHANGES_REQUESTED** [review]
Please, remove the nested class rules.

File: scss/components/_breadcrumbs.scss [hunk 1]
```diff
@@ -34,5 +25,14 @@
             font-size: 18px;
             padding: 0 8px;
         }
+
+        &--link {
```
Comment (vierno) [inline scss/components/_breadcrumbs.scss]: Link should be an element, not a modifier.

File: scss/components/_notification.scss [hunk 2]
```diff
@@ -35,11 +34,11 @@
         opacity: 1;
     }
 
-    &__text {
+    .bs-ui-notification__text {
```
Comment (vierno) [inline scss/components/_notification.scss]: This element should not be nested

File: scss/components/_table.scss [hunk 3]
```diff
@@ -87,81 +56,92 @@
...
+        &__link {
             color: $color-gray-6;
             display: block;
             margin: -10px -10px -10px -20px;
             padding: 10px 10px 10px 20px;
             text-decoration: none;
         }
 
-        tr {
+        &__tr {
```
Comment (vierno) [inline scss/components/_table.scss]: Could we use more meaningful names instead of the tags as classes?

File: scss/components/_table.scss [hunk 4]
```diff
@@ -87,81 +56,92 @@
...
             background-color: $color-gray-2;
             color: $color-gray-6;
             padding: 10px 10px 10px 20px;
+
+            &:hover {
+                background: $color-blue-lighter;
+            }
+
+            // FIXME
+            &.selected {
```
Comment (vierno) [inline scss/components/_table.scss]: Fix me?

**Review by vierno — CHANGES_REQUESTED** [review]

File: scss/components/_form-controls.scss [hunk 5]
```diff
@@ -8,15 +8,11 @@
 
 .bs-ui-switch {
     cursor: pointer;
+    font-family: $font-default;
+    font-size: $font-size-14;
+    letter-spacing: $letter-spacing-14;
```
Comment (marcelometal) [inline scss/components/_form-controls.scss]: should be used @extend %font-14?

**Review by marcelometal — CHANGES_REQUESTED** [review]

**Review by vierno — APPROVED** [review]

**Review by marcelometal — APPROVED** [review]

File: scss/components/_form_controls_input.scss [hunk 6]
```diff
@@ -30,6 +28,18 @@
         &:focus {
             border-color: $color-hit;
         }
+
+        &--textarea {
```
Comment (ricobl) [inline scss/components/_form_controls_input.scss]: This should be an element, not a modifier...

File: scss/components/_form_controls_input.scss [hunk 7]
```diff
@@ -30,6 +28,18 @@
...
+            &.bs-ui-form-control__field {
```
Comment (ricobl) [inline scss/components/_form_controls_input.scss]: Is it required to have two classes here?

File: scss/components/_options.scss [hunk 8]
```diff
@@ -46,7 +45,7 @@
         text-align: left;
         white-space: nowrap;
 
-        a {
+        &--link {
```
Comment (ricobl) [inline scss/components/_options.scss]: This should be an element, not a modifier.

File: scss/components/_table.scss [hunk 9]
```diff
@@ -7,72 +7,41 @@
...
 
-                a {
+                .bs-ui-table__link {
                     color: $color-gray-9;
                 }
 
-                a:before,
-                span:before {
+                .bs-ui-table__link:before,
+                .bs-ui-table__span:before {
```
Comment (ricobl) [inline scss/components/_table.scss]: Can we find a better name than `span`?

**Review by ricobl — CHANGES_REQUESTED** [review]
Hey @richardkevin! Can you please review my comments?

Also, looks like there is no change in dependencies but the `yarn.lock` has changed. I believe it should be unchangend in this PR.

**Pull request merged**